auto_nag/db.py

```diff
@@ -0,0 +1,422 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import calendar
+import csv
+import dateutil.parser
+from filelock import FileLock
+import json
+from libmozdata import utils as lmdutils
+import os
+import six
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy import create_engine
+from sqlalchemy.orm import relationship, sessionmaker
+from sqlalchemy import Column, ForeignKey, Integer, String
+
+from auto_nag import logger, utils
+from auto_nag.history import History
+
+
+Base = declarative_base()
+lock_path = utils.get_config('common', 'lock')
+db_path = utils.get_config('common', 'database')
+engine = create_engine(db_path)
+DBSession = sessionmaker(bind=engine)
+Base.metadata.bind = engine
+session = DBSession()
```

Comment:
That part seems really complex, there must be a helper to open a session somewhere

Reply:
I didn't find anything for that, if you've any clues...

Reply:
Nah you're right, it does not get [much simpler](https://docs.sqlalchemy.org/en/13/orm/session_basics.html)